src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/EventsRepository.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Repository
+{
+    /// <summary>
+    /// Handles events repository. Notice that the all methods should modify the Subject attribute of the
+    /// CloudEvent, since cosmosDb fails if Subject contains slashes '/'.
+    /// </summary>
+    public class EventsRepository : IEventsRepository
+    {
+        private readonly ILogger _logger;
+        private readonly Uri _databaseUri;
```

Comment:
This is used only by the constructor and can be made a local variable instead.

Reply:
Removed the variable and used the value from database instead .